company-ctags.el

```diff
@@ -347,6 +377,8 @@ Execute COMMAND with ARG and IGNORED."
                  (company-ctags-buffer-table)
                  (or (company-grab-symbol) 'stop)))
     (candidates (company-ctags--candidates arg))
+    (match (company-ctags--match-bounds (company-grab-symbol) arg))
```

Comment:
I don't know if calling `company-grab-symbol` again is a good idea (we should already have the prefix). I'll see what other people say on https://github.com/company-mode/company-mode/issues/948.

Reply:
I switched to text property for this. It is said to be the common practice.